README.md

```diff
@@ -1 +1,94 @@
-# starkwager-backend
\ No newline at end of file
+StarkWager
+StarkWager is a decentralized application (dApp) designed to leverage the StarkNet ecosystem, enabling users to place wagers on simple agreements or competitions with minimal transaction fees. The app ensures a seamless experience for users to deposit, withdraw, and interact with wagers, making it a powerful tool in the StarkNet ecosystem.
+
+🌟 Features
+Create Wallet: Get a wallet that can be funded and used to place a wager 🚧
+
+Connect Existing Wallet: Easily fund wagers from already existing wallets (e.g., Argent) 🚧
+
+Create/Join Wagers: All operations are handled through smart contracts 🚧
+
+Invite Others: Invite other users to join a wager 🚧
+
+Fair Distribution: Rewards are distributed proportionally based on stake 🚧
+
+🛠 Technology Stack
+Smart Contracts: Cairo (StarkNet's native smart contract language)
+
+Blockchain: StarkNet
+
+Frontend: Next.js
+
+Testing Framework: snforge (for Cairo smart contracts)
+
+Getting Started
+To get started with the StarkWager App, follow the instructions below:
+
+1. Install Dependencies
+The project uses pnpm as the package manager. First, install pnpm globally:
```

Comment:
This documentation is for the smart contract side, so there's no need for steps of installing `pnpm` here